app/controllers/import_controller.rb

```diff
@@ -178,7 +178,10 @@ def import_sdx_field(patient, field, row, row_ind, col_num)
       patient[field] = import_field(field, Date.strptime(row[col_num], '%m%d%Y'), row_ind)
     elsif %i[primary_telephone secondary_telephone alternate_primary_telephone].include?(field)
       phone = Phonelib.parse(row[col_num], 'US')
-      patient[field] = phone.full_e164 || row[col_num] && return unless phone.full_e164.blank? || phone.full_e164.sub(/^\+1+/, '').length != 10
+      unless phone.full_e164.blank? || phone.full_e164.sub(/^\+1+/, '').length != 10
+        patient[field] = phone.full_e164 || row[col_num]
+        return
+      end
 
```

Comment:
I don't know if this is more readable without the return (Note: code untested)

`if phone.full_e164.present? || phone.full_e164.sub(/^\+1+/, '').length == 10
    patient[field] = phone.full_e164 || row[col_num]
else
  patient[:alternate_international_telephone] = row[col_num] if field == :alternate_primary_telephone
  patient[:international_telephone] = row[col_num] if field == :primary_telephone
end`

Reply:
I do agree with you, but after refactoring, rubocop takes issue with the level of nesting (since the `if field == :secondary_telephone` block below also has to be nested in the `else` case). So I have inverted the `unless` statement, but left the `return`.